src/surabhi/WhileNumberRange.java

```diff
@@ -0,0 +1,87 @@
+package surabhi;
+
+public class WhileNumberRange {
+
+	void findEvenNumber(int startIndex, int endIndex) {
+
+		if(startIndex>=0 && startIndex<endIndex) {
+			System.out.println("Even Numbers are:-");
+			int index=startIndex;
```

Comment:
you don’t require to create extra variable, you can handle this without **index**, think on this

please refer below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/219/files